tests/test_milvus_vector_db_service.py

```diff
@@ -366,82 +364,24 @@ def test_delete(milvus_service: MilvusVectorDBService, idx_part_collection_confi
     milvus_service.drop(collection_name)
 
 
-@pytest.mark.milvus
-def test_single_instance_with_collection_lock(milvus_service: MilvusVectorDBService,
```

Comment:
Question: Were these tests deleted in a previous PR?

Reply:
It was pushed to `branch-23.11` but not update in the `fea-sherlock`